pontoon/base/views.py

```diff
@@ -210,7 +210,7 @@ def _get_entities_list(locale, project, form):
     }, safe=False)
 
 
-def _get_all_entities(locale, project, form, entities):
+def _get_all_entities(request, locale, project, form, entities):
```

Comment:
Why are you adding `request` here? You don't use it in the function.

Reply:
What goes for above will be used here, but since above code was too complex I held it for the time we can come up with something simpler and better.